dll/cpl/sysdm/general.c

```diff
@@ -62,29 +62,29 @@ static VOID InitLogo(HWND hwndDlg)
     BITMAP logoBitmap;
     BITMAP maskBitmap;
     BITMAPINFO bmpi;
-    HDC hDC = GetDC(hwndDlg);
+    HDC hDC;
```

Comment:
Bug?

Reply:
Oups, my mistake when taking previous batch of comments. I fix